tests/ipv4/test_ip_priority_field.py

```diff
@@ -12,26 +8,19 @@ def test_counter_ip_dscp(api, b2b_raw_config, utils):
       against expected
     """
     f = b2b_raw_config.flows[0]
-    f.packet = [
-        flow.Header(flow.Ethernet()),
-        flow.Header(flow.Ipv4())
-    ]
-    utils.apply_config(api, b2b_raw_config)
+    f.packet.ethernet().ipv4()
+    ipv4 = f.packet[1]
     phb = ['PHB_DEFAULT'] + ['PHB_CS%d' % i for i in range(1, 8)] + \
           ['PHB_AF%d' % i for j in range(11, 51, 10) for i in range(j, j + 3)]
     phb = phb + ['PHB_EF46']
     af_ef = [
         '10', '12', '14', '18', '20', '22', '26',
         '28', '30', '34', '36', '38', '46'
     ]
-
     for i, p in enumerate(phb):
-        dscp = flow_ipv4.Dscp(
-            phb=flow.Pattern(getattr(flow_ipv4.Dscp, p))
-        )
-        prio = flow_ipv4.Priority(dscp)
-        f.packet[1].ipv4.priority = prio
-        utils.apply_config(api, b2b_raw_config)
+        ipv4.priority.choice = 'dscp'
```

Comment:
we don't need to set a choice here.

Reply:
directly setting phb value throwing attribute error
>>> print(i)
{}
>>> i.priority.dhcp.phb.value = 10
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'FlowIpv4Priority' object has no attribute 'dhcp'
>>>

Reply:
it should be `dscp`.